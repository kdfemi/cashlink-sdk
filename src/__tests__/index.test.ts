import CashlinkSdk from '../index';

describe('Test sdk index', (): void => {
  test('open', (): void => {
    expect(CashlinkSdk).toMatchSnapshot();
  });

});