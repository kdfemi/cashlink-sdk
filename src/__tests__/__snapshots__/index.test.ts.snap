// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test sdk index open 1`] = `
{
  "init": [Function],
  "profile": [Function],
  "transact": [Function],
  "transactExternal": [Function],
}
`;
